recipes/gst-plugins-base/all/conanfile.py

```diff
@@ -0,0 +1,217 @@
+            self.build_requires("pkgconf/1.7.3")
+        if self.settings.os == 'Windows':
+            self.build_requires("winflexbison/2.5.22")
+        else:
+            self.build_requires("bison/3.5.3")
+            self.build_requires("flex/2.6.4")
+        if self.options.with_introspection:
+            self.build_requires("gobject-introspection/1.68.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_meson(self):
+        defs = dict()
+
+        def add_flag(name, value):
+            if name in defs:
+                defs[name] += " " + value
+            else:
+                defs[name] = value
+
+        def add_compiler_flag(value):
+            add_flag("c_args", value)
+            add_flag("cpp_args", value)
+
+        def add_linker_flag(value):
+            add_flag("c_link_args", value)
+            add_flag("cpp_link_args", value)
+
+        meson = Meson(self)
+        if self.settings.compiler == "Visual Studio":
+            add_linker_flag("-lws2_32")
+            add_compiler_flag("-%s" % self.settings.compiler.runtime)
+            if int(str(self.settings.compiler.version)) < 14:
+                add_compiler_flag("-Dsnprintf=_snprintf")
+        if self.settings.get_safe("compiler.runtime"):
+            defs["b_vscrt"] = str(self.settings.compiler.runtime).lower()
+        defs["tools"] = "disabled"
+        defs["examples"] = "disabled"
+        defs["benchmarks"] = "disabled"
+        defs["tests"] = "disabled"
+        defs["wrap_mode"] = "nofallback"
+        defs["introspection"] = "enabled" if self.options.with_introspection else "disabled"
+        defs["gl"] = "enabled" if self.options.with_gl else "disabled"
+        meson.configure(build_folder=self._build_subfolder,
+                        source_folder=self._source_subfolder,
+                        defs=defs)
+        return meson
+
+    def _copy_pkg_config(self, name):
+        root = self.deps_cpp_info[name].rootpath
+        pc_dir = os.path.join(root, 'lib', 'pkgconfig')
+        pc_files = glob.glob('%s/*.pc' % pc_dir)
+        if not pc_files:  # zlib store .pc in root
+            pc_files = glob.glob('%s/*.pc' % root)
+        for pc_name in pc_files:
+            new_pc = os.path.basename(pc_name)
+            self.output.warn('copy .pc file %s' % os.path.basename(pc_name))
+            shutil.copy(pc_name, new_pc)
+            prefix = tools.unix_path(root) if self.settings.os == 'Windows' else root
+            tools.replace_prefix_in_pc_file(new_pc, prefix)
+
+    def build(self):
+        for p in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**p)
+
+        self._copy_pkg_config("glib")
+        self._copy_pkg_config("gstreamer")
+        with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
```

Comment:
I haven't checked, but is the .pc handling still actual for conan-center?
```suggestion
        return meson

    def build(self):
        for patch in self.conan_data.get("patches", {}).get(self.version, []):
            tools.patch(**patch)

        with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
```